app/subscriber/src/features/login/BrowserLogin.tsx

```diff
@@ -41,60 +43,116 @@ export const BrowserLogin: React.FC<IBrowserLoginProps> = ({ login }) => {
                   </div>
                 </div>
+                <div className="footer" onClick={() => login()}>
+                  <b>Copyright info:</b>
+                  <p>
+                    This account grants you access to copyrighted material for your own use. It does
+                    not grant you permission to fix, copy, reproduce or archive any of the material
+                    contained within. <br /> <br />
+                    You cannot redistribute this information to anyone without violating your
+                    copyright agreement.
+                  </p>
+                </div>
               </Col>
-            </div>
-          </Col>
-          <Col className="system-message-box">
-            <div className="system-message-containing-box">
               <Show visible={!!systemMessage?.message && systemMessage.isEnabled}>
-                <p>{parse(systemMessage?.message ?? '')}</p>
+                <Col className="system-message-box">
+                  <div className="system-message-containing-box">
+                    <p>{parse(systemMessage?.message ?? '')}</p>
+                  </div>
+                </Col>
               </Show>
             </div>
           </Col>
         </Row>
       </Col>
-
+      <Show visible={showModal}>
+        <div id="myModal" className="modal">
+          <div className="modal-content">
+            <span
+              className="close"
+              onClick={() => {
+                setShowModal(false);
+              }}
+            >
+              &times;
+            </span>
+            <div>
+              <p>
+                <b>Thank you for your interest in Media Monitoring Insights.</b>
+              </p>
+              <p>
+                <b>Key features:</b>
+              </p>
+              <ul>
+                <li>Aggregation of newspapers, radio shows, and online articles</li>
+                <li>Transcription services</li>
+                <li>BC’s top stories as they break</li>
+                <li>Articles related to major stories</li>
+              </ul>
+              <p>
+                Access to this service is limited to Government of British Columbia staff as per our
+                agreements with our media suppliers.
+              </p>
+              <br />
+              <p>
+                <b>
+                  If you are eligible for access, email{' '}
+                  <a href="mailto:Scott.Ryckman@gov.bc.ca">Scott.Ryckman@gov.bc.ca</a> to get set
+                  up.
+                </b>
+              </p>
+              <br />
+              <p>
+                Note: <br />
+                There is a cost recovery charge in excess of $4,500 annually, please ensure that you
+                have authorization for this purchase prior to emailing.
+              </p>
+            </div>
+          </div>
+        </div>
+      </Show>
       <img src="/assets/mm_logo.svg" alt="MM Logo" className="mm-logo" />
```

Comment:
Think this is supposed to be removed - can confirm with Bobbi!

Reply:
I'll check with her.